src/iperf_api.c

```diff
@@ -1835,6 +1851,20 @@ iperf_parse_arguments(struct iperf_test *test, int argc, char **argv)
 	i_errno = IEUDPBLOCKSIZE;
 	return -1;
     }
+
+#ifdef HAVE_UDP_SEGMENT
+    if (test->protocol->id == Pudp && test->settings->gso) {
+        test->settings->gso_dg_size = blksize;
+        /* use the multiple of datagram size for the best efficiency. */
+        if (test->settings->gso_dg_size > 0) {
+            test->settings->gso_bf_size = (test->settings->gso_bf_size / test->settings->gso_dg_size) * test->settings->gso_dg_size;
+        } else {
+            /* If gso_dg_size is 0 (unlimited bandwidth), use default UDP datagram size */
+            test->settings->gso_dg_size = 1472; /* Standard UDP payload size for Ethernet MTU */
```

Comment:
Why not using `DEFAULT_UDP_BLKSIZE` instead of 1472?  If it is important that 1472 will be used, suggest to add `#define DEFAULT_GSO_DG_BLKSIZE 1472`.